tests/yam/agama/auto/auto.pm

```diff
@@ -0,0 +1,23 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi;
```

Comment:
better to import only 'assert_screen' from testapi, not the whole one so we minimize the use of testapi as pattern.